_site/feed.xml

```diff
@@ -4,7 +4,7 @@
     <subtitle>A Blog by Josh Adams</subtitle>
     <link href="http://racecondition.software/feed.xml" rel="self" />
     <link href="http://racecondition.software" />
-    <updated>2018-04-08T16:56:34-07:00</updated>
+    <updated>2018-04-15T19:03:07-07:00</updated>
```

Comment:
Why this change?

Reply:
Ah, this is because you check-in the `_site/` directory. So this is an artifact of running `jekyll build`.

From the original PR description: 

> I noticed you have `_site/` checked in. Usually you want this to be ignored. Was that intentional?

You probably want `_site/` to be in your `.gitignore`. But, it might depend on your hosting setup.

Reply:
Oh, I just realized you don't have a `.gitignore` 😊 

I'd recommend adding one. 😄

Reply:
I will do so.